static/src/stylesheets/module/content-garnett/_article-immersive.scss

```diff
@@ -380,7 +261,7 @@
 
     .element-pullquote.element--showcase {
         @include mq(tablet) {
-            margin-bottom: $gs-baseline * 3;
+            // margin-bottom: $gs-baseline * 3;
```

Comment:
Can we delete this, not just comment it out?

Reply:
Done.